xml/System/Double.xml

```diff
@@ -3232,7 +3232,7 @@ If a separator is encountered in the `s` parameter during a parse operation, and
   
  The return value can be <xref:System.Globalization.NumberFormatInfo.PositiveInfinitySymbol%2A>, <xref:System.Globalization.NumberFormatInfo.NegativeInfinitySymbol%2A>, <xref:System.Globalization.NumberFormatInfo.NaNSymbol%2A>, or a string of the form:  
   
- [sign]integral-digits[.[fractional-digits]][e[sign]exponential-digits]  
+ [sign]integral-digits[.[fractional-digits]][E[sign]exponential-digits]  
```

Comment:
This is specifically for the `ToString()` overload that takes no parameters, correct?

Reply:
This explanation is present in two overloads:
![image](https://user-images.githubusercontent.com/12971179/70343621-9b283880-180c-11ea-9196-1021424b4684.png)

Reply:
Then LGTM.

I just wanted to ensure this wasn't being done on any method that took a `string format`, as that will use `E` for an uppercase format specifier (such as `G`, `R`, `F`, `N`, or `E`) and `e` for a lowercase format specifier (such as `g`, `r`, `f`, `n`, or `e`).